script.js

```diff
@@ -0,0 +1,13 @@
+function shifumi() {
+
```

Comment:
You can use the indentation or line break rules you want, but try to be consistant. Why having a line break here at the top?

Reply:
it's about visibility, find it better but I have to think that other people will start to read my code